sql/2019/12_Mobile_Web/12_11.sql

```diff
@@ -0,0 +1,9 @@
+#standardSQL
+
+# links or buttons only containing an icon
+
+SELECT
+    COUNT(url) AS total,
+    ROUND(COUNTIF(JSON_EXTRACT_SCALAR(JSON_EXTRACT_SCALAR(payload, '$._almanac'), "$['12.11']") > 0) * 100 / COUNT(0), 2) AS pct_has_icon_button
```

Comment:
Getting this error: 

> Error: No matching signature for operator > for argument types: STRING, INT64. Supported signatures: ANY > ANY at [7:19]

Casting to an int doesn't seem to help, I think because of malformed results for some pages. (probably related to [this issue](https://discuss.httparchive.org/t/representative-urls-for-common-unusual-markup-for-investigation/1706/5))

So let's go with a UDF for this query.